pixel/patterns/post-list.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Title: Posts list
+ * Slug: pixel/posts-list
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:query {"query":{"perPage":"10","pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"tagName":"main","displayLayout":{"type":"list","columns":3},"layout":{"type":"constrained"}} -->
+<main class="wp-block-query"><!-- wp:post-template -->
+<!-- wp:group {"layout":{"type":"constrained"},"style":{"border":{"width":"2px"},"spacing":{"padding":{"top":"33px","right":"33px","bottom":"33px","left":"33px"}}},"borderColor":"primary","className":"pixel-shadow"} -->
+<div class="wp-block-group pixel-shadow has-border-color has-primary-border-color" style="border-width:2px;padding-top:33px;padding-right:33px;padding-bottom:33px;padding-left:33px"><!-- wp:post-date {"style":{"typography":{"textTransform":"uppercase","fontSize":"12px"}}} /-->
```

Comment:
use the padding from spacers `--wp--preset--spacing--60` (large in UI)

Reply:
Done in [c48daa6](https://github.com/Automattic/themes/pull/6481/commits/c48daa6b85598caae60b42a25c1f40659b275d66).